source/GameData.cpp

```diff
@@ -1136,6 +1155,19 @@ void GameData::LoadFile(const string &path, bool debugMode)
 			for(const DataNode &child : node)
 				list.push_back(child.Token(0));
 		}
+		else if(key == "category" && node.Size() >= 2)
+		{
+			vector<string> &categoryList = categories[node.Token(1)];
+			for(const DataNode &child : node)
+			{
+				// If a given category already exists, it will be
+				// moved to the back of the list.
+				const auto &it = find(categoryList.begin(), categoryList.end(), child.Token(0));
```

Comment:
This binds a reference to an rvalue (the result of `find`). Instead, just accept the value:
```suggestion
				const auto it = find(categoryList.begin(), categoryList.end(), child.Token(0));
```